### PR TITLE
Sonarqube

### PR DESCRIPTION
**Pull request by ahoffer**
Circle CI did not pick up previous PR. Opening a new one to attempt to trigger Circle CI.

### CODE REVIEW

**Comment by ahoffer** [comment]
In retrospect, I got a little carried away. Probably best if I just stop changing things.

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,53 +2,80 @@ version: 2.1
...
       - image: circleci/openjdk:11-jdk
     steps:
+      - restore_cache: *restore_cache
       - checkout
       - run:
           name: OWASP
           command: |
-            ./gradlew dependencyCheckAnalyze --info
-  Scan Docker Image:
+            ./gradlew dependencyCheckAnalyze -x test
```
Comment (jhunzik) [inline .circleci/config.yml]: ❓ Find/Replace mistake? The OWASP task doesn't run the test task.

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,53 +2,80 @@ version: 2.1
...
+  Anchore:
```
Comment (jhunzik) [inline .circleci/config.yml]: I'm conflicted, but this matches up better with the Sonar and OWASP

File: .circleci/config.yml [hunk 3]
```diff
@@ -2,53 +2,80 @@ version: 2.1
...
           paths:
             - ion-search.tar
+  Test:
+    machine: *virtual_machine
+    steps:
+      - run: *install_java
+      - run: *configure_java
+      - checkout
+      - restore_cache: *restore_cache
+      - run: ./gradlew check
```
Comment (emmberk) [inline .circleci/config.yml]: ✏️Doesn't the `check` task run the `assemble` task? Looks like it might not make sense to break out the tasks this way.

**Comment by ahoffer** [comment]
Merged a different branch instead of this one. Closing the PR.

**Pull request closed without merging**